app/V1Module/presenters/RegistrationPresenter.php

```diff
@@ -103,7 +103,7 @@ public function actionCreateAccount() {
 
     // check if the email is free
     $email = trim($req->getPost("email"));
-    if ($this->users->getByEmail($email) !== null) {
+    if ($this->logins->getByUsername($email) !== null) {
```

Comment:
This is somewhat strange -- getByUsername($email). This method should have a better name, or an explanatory comment is required.

Reply:
I do not follow... this is classical repository method which finds a login based on given username... what do you find strange about it?

Reply:
When you say user name, first that comes to mind is firstName and lastName. getByLogin or getByLoginName would be much better.

Reply:
But the column is `username`, therefore, repository convention says to have this method name...

Reply:
And no... we are not gonna rename the database column...

Reply:
OK, can you at least leave a note (in a comment) that username is in fact login?

Reply:
Can do